app/assets/javascripts/initializers/initializeArticleReactions.js

```diff
@@ -27,6 +27,25 @@ function setReactionCount(reactionName, newCount) {
   }
 }
 
+function setSumReactionCount(counts) {
+  let totalCountObj = document.getElementById('reaction_total_count');
+  if (totalCountObj && counts.length > 2) {
```

Comment:
Not a blocker, but to inform future refactoring here, is there a reason we prefer client-side math here over adding a `total` key to the `ReactionsController#index` payload?

Reply:
Yeah @jaw6 I had the same thought. My approach here was to just get this working with the existing API, and treating that as the less intrusive approach with so many moving pieces as we get this live, but definitely seems like something we could just add to the request later.

Reply:
I probably have some scars from [working on count consistency on multiple reactions for a bit](https://github.com/forem/forem/pull/18808#issuecomment-1363129175), but I'm very nervous about doing this math, specifically, on the front-end. I want to believe it can work, it seems to work, I don't want to block this PR, but I have to say it does _not_ feel like we're _minimizing_ the moving pieces by introducing a new count system here.